README.md

```diff
@@ -1,66 +1,166 @@
+#### Build a REST API with Node JS and Express
 
-This is a companion repository for: "[Quickstart in Couchbase with Node JS and Express](https://developer.couchbase.com/tutorial-quickstart-nodejs/)" at [developer.couchbase.com](https://developer.couchbase.com), which aims to get you up and running with Couchbase and the [NodeJS SDK](https://docs.couchbase.com/nodejs-sdk/current/hello-world/start-using-sdk.html). Connect to a Couchbase cluster, create, read, update, and delete documents, and learn how to write simple parameterized SQL++ queries.
+Often, the first step developers do after creating their database is to create a REST API that can perform Create, Read, Update, and Delete (CRUD) operations for that database. This repo is designed to teach you and give you a starter project (in Node JS using Express) to generate such a REST API. After you have installed travel-sample bucket in your database, you can run this application which is a REST API with Swagger documentation so that you can learn:
 
-We will be using the latest version of Couchbase (version 7) that enables scopes and collections. For the easiest setup experience, we recommend trying Couchbase Capella, our fully-managed DBaaS offering. [Claim your free trial!](https://cloud.couchbase.com/sign-up)
+1. How to create, read, update, and delete documents using [Key Value operations](https://docs.couchbase.com/nodejs-sdk/current/howtos/kv-operations.html) (KV operations). KV operations are unique to couchbase and provide super fast (think microseconds) queries.
+2. How to write simple parametrized [SQL++ queries](https://docs.couchbase.com/nodejs-sdk/current/howtos/n1ql-queries-with-sdk.html) using the built-in travel-sample bucket.
+
+Full documentation for the tutorial can be found on the [Couchbase Developer Portal](https://developer.couchbase.com/tutorial-quickstart-nodejs).
+
+
+If you are looking for a quickstart using [Ottoman](https://ottomanjs.com/), you can find it in this [repo](https://github.com/couchbase-examples/ottomanjs-quickstart).
 
-Alternatively, you can [install Couchbase with docker](https://docs.couchbase.com/server/current/getting-started/do-a-quick-install.html) or [directly on your local device](https://docs.couchbase.com/server/current/install/install-intro.html). 
 
 ## Prerequisites
 
 To run this prebuilt project, you will need:
 
-- A Couchbase Capella cluster or Couchbase 7 installed locally
-- NodeJS & NPM (v12+)
-- Code Editor
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/python-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+  - To run this tutorial using a self managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
+- [Node js](https://nodejs.org/en/download) installed on local machine.
```

Comment:
Can you also add the minimum version of Node that is supported?

Reply:
Added the minimum version of node based on the couchbase sdk package.json configuration.